src/EbmlSInteger.cpp

```diff
@@ -70,23 +70,23 @@ std::uint64_t EbmlSInteger::UpdateSize(const ShouldWrite & writeFilter, bool /*
   if (!CanWrite(writeFilter))
     return 0;
 
-  const auto Value = GetValue();
-  if (Value <= 0x7F && Value >= (-0x80)) {
+  const auto value = GetValue();
```

Comment:
I really like avoiding shadowing variables, so thanks for the patch. That being said, I really dislike using a name that only differs in case from an existing name. Can you please use anything other than `value`? Don't have a really great suggestion. Maybe `Val or `StoredValue` or even `Cmp` (for all the comparisons).

Reply:
`val` will do